$OpenBSD$

Use the crypton-era ASN.1 packages and time-hourglass in the vendored
simplexmq dependency set.

--- simplexmq.cabal.orig
+++ simplexmq.cabal
@@ -335,8 +335,8 @@
       crypto
   build-depends:
       aeson ==2.2.*
-    , asn1-encoding ==0.9.*
-    , asn1-types ==0.3.*
+    , crypton-asn1-encoding ==0.10.*
+    , crypton-asn1-types ==0.4.*
     , async ==2.2.*
     , attoparsec ==0.14.*
     , base >=4.14 && <5
@@ -352,12 +352,12 @@
     , data-default ==0.7.*
     , directory ==1.3.*
     , filepath ==1.4.*
-    , hourglass ==0.2.*
+    , time-hourglass ==0.3.*
     , http-types ==0.12.*
     , http2 >=4.2.2 && <4.3
     , iproute ==1.7.*
     , iso8601-time ==0.1.*
     , memory ==0.18.*
     , mtl >=2.3.1 && <3.0
     , network >=3.1.2.7 && <3.2
     , network-info ==0.2.*
